### DISCUSSION TITLE
Unable to continue application after selecting school

### DISCUSSION THREAD

**Discussion by Gingeh** [community/community#128947] [GitHub Education]
### Hi everyone,

On https://education.github.com/discount_requests/application, I've logged into my account, selected "Student", added and verified my university email, and entered the name of my university into the text input. Nothing happens when I press "Continue" and give access to my location. I haven't gotten any error messages. I have tried and failed to apply on multiple devices and multiple networks.

**Comment by buu-huu** [comment]
Is the "continue" button greyed out?

**Reply by Gingeh** [reply]
before pressing continue
![image](https://github.com/community/community/assets/39150378/b416bbed-44fa-443f-b6fb-e99a1d297c02)

after pressing continue
![image](https://github.com/community/community/assets/39150378/38ebfb5b-eedf-405d-a17b-90e3b01d268b)

**Reply by buu-huu** [reply]
Can you check, if you allowed access to your location on this website? It's necessary to do not block location the location service for this webpage.

**Reply by Gingeh** [reply]
![image](https://github.com/community/community/assets/39150378/e037fc0d-80e5-4818-a324-373956b8cb10)

**Comment by davevad93** [comment]
Hi @Gingeh ,

I noticed that you have the `PRO` badge, which means that you have probably been approved for the `Student Developer Pack`.
To be sure you could login to `Copilot` in `VS Code` with your GitHub account, if you can use it normally it means your benefits are fully available.

**Reply by Gingeh** [reply]
I am trying to renew my benefits as they will expire this month

**Reply by davevad93** [reply]
Oh, I see, then I don't know why the application page behaves that way, If you're still having issues you could try contact support by opening a ticket [here](https://support.github.com/contact/education).

**Reply by Gingeh** [reply]
I'm not sure which topic would be the most relevant for the support ticket. Maybe "I have found a technical bug", but when I select that it just tells me to make a post on the community discussions (here) instead.
![image](https://github.com/community/community/assets/39150378/ac74c520-f2fa-458e-8d3c-21e15dd4f723)

**Reply by davevad93** [reply]
Mmm, in that case maybe you should open a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Reply by davevad93** [reply]
Hopefully someone in charge of `GitHub Education` will answer you here, but I would also try opening a ticket to support, just in case.

**Reply by Gingeh** [reply]
okay, I will try that

**Comment by Moerliy** [comment]
I ran into the same thing too. 
I have had education in the past, and it expired, but I'm still legible. 
I can't continue the application with the same problem.

Did you manage to find help via the support? @Gingeh

**Reply by Gingeh** [reply]
I'm so sorry I don't remember how I was able to resolve the issue :(

**Reply by joco163** [reply]
> I ran into the same thing too. I have had education in the past, and it expired, but I'm still legible. I can't continue the application with the same problem.
> 
> Did you manage to find help via the support? @Gingeh

Hi,
i am dealing with the exact same problem. Did anyone find a solution?
KR

**Reply by Moerliy** [reply]
I played around with some browsers and in safari it finally worked.

**Reply by joco163** [reply]
yes thanks. it worked for me aswell in Chrome on my Smartphone.
